.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: php
-dist: trusty
-
+services:
+  - docker
```

Comment:
can you please explain why this is necessary...i'd really like to test as "normal" as possible and not using any docker mysql stuff...

Reply:
Well, that is necessary because you do not only want to test against a single version of MySQL. You don't even want to test against MySQL (the Oracle/SUN one) alone, but also against MariaDB. Spinning up different docker instances with different mysql/mariadb versions allows you exactly that.

> Update Travis to run tests with php7.3 and with mysql 5.6, 5.7 and 8.0 and mariadb 10.3 and 10.4.

Reply:
well but it acutally does only test using mariadb 10.4 doesn't it?

Reply:
> well but it acutally does only test using mariadb 10.4 doesn't it?

I see you understood the `matrix` object :)
At the moment it's configured for 10.4, because I did not want to spin up 5 jobs per commit while I am testing the changes in the yaml. But when I'm done, I'd reenable the tests.

Reply:
Oh gee, yeah, totally not seen the `matrix` tag - sorry :P nicely done